app/src/ui/lib/filter-list.tsx

```diff
@@ -372,6 +376,13 @@ export class FilterList<T extends IFilterListItem> extends React.Component<
     }
   }
 
+  private onEnterClicked = (text: string) => {
+    const rows = this.state.rows.length
+    if (rows === 0 && this.props.onEnterClicked) {
```

Comment:
We prefer to use explicit checks for `undefined`, and I think we should also check that `text` has something in it:
```suggestion
    if (rows === 0 && text.trim().length > 0 && this.props.onEnterClicked !== undefined) {
```